src/App.jsx

```diff
@@ -14,70 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, chahgeSelectedGood] = useState('Jam');
+  const [title, setTitle] = useState(`${selectedGood} is selected`);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {title}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        <button
+          data-cy="ClearButton"
+          type="button"
+          className="delete ml-3"
+          onClick={() => {
+            chahgeSelectedGood('');
+            setTitle('No goods selected');
+          }}
+        />
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        {goods.map(good => (
+          <tbody>
```

Comment:
there should be only one tbody on the page, inside of which you should map <tr>
also where is `key` prop?